specs/pvs/corla.pvs

```diff
@@ -1499,7 +1619,9 @@ END authentication
 
 % @relation back_end inherit system_architecture_component
 % @relation front_end inherit system_architecture_component
-system_architecture_component: DATATYPE WITH SUBTYPES back_end, front_end
+system_architecture_component: DATATYPE
+  WITH SUBTYPES back_end,
+                front_end
 BEGIN
   IMPORTING information_systems, database, server
 
```

Comment:
line 1578/1700 "Minimum cardinality" -- for readability, rewrite: "At least two of each are necessary because..."